src/components/Translator.tsx

```diff
@@ -31,7 +32,7 @@ function Translator<T extends TranslatorType = 'google'>(
 
   const onMessage = useCallback((event: WebViewMessageEvent) => {
     const result = event.nativeEvent.data;
-    if (!result || result === LOADING_MESSSAGE) {
+    if (!result || result === LOADING_MESSSAGE || result === 'Enter a URL') {
```

Comment:
I haven't reproduced this issue yet, and I need test code, so can you please work on another PR?
```suggestion
    if (!result || result === LOADING_MESSSAGE) {
```

Reply:
ok sure

Reply:
@basitmir @KoreanThinker 

I receive the same result message but it was on system language of my phone.
So I think we need to extend the `source` option in WebView to ignore phone system language:
```ts
      source={{
        uri,
        headers: {
          'Accept-Language': 'en',
        },
      }}
```

Reply:
@KoreanThinker I can reproduce this result (`Enter a URL`) when I have high ping to the Google Translate service (WebView have not enough time to render the page)